embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/worker/Worker.kt

```diff
@@ -53,5 +53,10 @@
          * Monitor thread that checks the main thread for ANRs.
          */
         object AnrWatchdogWorker : Background("anr-watchdog")
+
+        /**
+         * Delivery Worker
+         */
+        object DeliveryWorker : Background("delivery")
```

Comment:
This is the worker that runs jobs that queue up the network request send and blocks on execution, waiting for a response.

Reply:
Ooh good call. Will add a TODO